content/posts/5-grzechow-juniora.mdx

```diff
@@ -0,0 +1,68 @@
+po to by ułatwić nam zycie, ale kazda kolejna biblioteka w projekcie to coraz większy próg wejścia
+dla nowych osób, tak więc zastanów sie dwa razy zanim sie pochwalisz wiedzą o jakiejś turbo nowej
+mega bibliotece i spróbujesz wrzucisz ją do projektu.
+
+## Nie wyciaganie wniosków z Code Review
+
+Tak! Tacy juniorzy tez sie niestety zdarzają. Od razu tłumaczę, ze nie chodzi mi o całkowite
+zignorowanie CR i dalsze popełnianie tych samych błędów, a po prostu o przeświadczenie juniora, ze
+rozwiązanie bardziej doświadczonego kolegi jest gorsze niz jego bo zobaczył je u jakiegoś
+wielkiego youtubera. Nikt z nas nie jest nieomylny tak więc nie popadajmy ze skrajności w
+skrajność, z robienia jak youtuber do robienia jak osoba z zespołu. Pamiętaj, kazde Code Review to kolejny
+krok w celu awansu twojej wiedzy. Kazdy komentarz wez sobie głęboko pod analize i staraj sie
+zrozuzmieć, dlaczego X, a nie Y. To jeden z najlepszych sposobów na pozyskanie doswiadczenia i
+wiedzy.
+
+## Brak znajomości narzedzi do debuggowania
+
+Przejdzmy teraz do bardziej technicznego tematu. Na początku drogi kazdy z nas jest skupiony na
+jak najlepszym poznaniu technologii w której pisze. Czy to C# czy JS czy jakikolwiek inny język to
+jednak najczęściej wybieranym sposobem "debuggowania" aplikacji jest CDD czyli Console Driven
+Development. No nie ma co się kłócić w prostych przypadkach jest to dość szybkie rozwiązanie
+jezeli chcemy sprawdzic co sie znajduje w X zmiennej. Jednak w przypadku juz bardzo rozległych
+aplikacji wiedza o tym jak uzywac debuggera jest wymagana! Dlatego nie pomijaj etapu jego nauki,
+gdyz szybko ci ona odda :D
+
+## Klepacz kodu
+
+Pewnie nie raz słyszałeś jak kogoś nazywano "klepaczem kodu". Kim jest taka osoba? W skrócie to po prostu programista, który niezbyt interesuje się rozwojem projektu, jest zazwyczaj mało pomysłowy na temat nowych dodatków do aplkacji, jego zmiany w kodzie często nie są jakkolwiek przemyślane, z code review wyciąga jedynie approve by móc zmerge'owac pull requesta i odpalić kolejnego taska, a jego celem jest po prostu przyjść do pracy wyklepać jak najszybciej zadania i do domu. Takie podejście bardzo szybko odbije się na twoim rozwoju. Staraj się poznawać projekt w którym jesteś nie tylko od strony technicznej ale równie biznesowej. Zadawaj pytania i staraj się samemu udzielać gdy widzisz taką mozliwosc, nikt ci nic nie zrobi za to, ze chciałeś dowiedzieć sie czegoś więcej o własnym "dziecku", które ty rozwijasz!
+
+<Image src="/images/typing-code.gif" alt="typing code" />
+
+## Podsumowanie
+
+Tych juniorskich "grzeszków" pewnie dałoby radę znaleźć o wiele wiecej, ale własnie ta 5 to te grzechy, które mi oraz moim znajomym(**serdecznie dziękuje społeczności Frontlive oraz Type of Web za udzielenie odpowiedzi na moje pytania**) najczęściej wpadały w oczy. Tak więc juniorze, trzymaj się mocno bo ta przygoda nie będzie prosta, ale jak nie popełnisz wyzej wymienionych błędów to na pewno będzie mniej stresująca. AUUUUU!
```

Comment:
ale ta właśnie pięć to te grzechy które mi

boziu jak to się źle czyta 😅

Reply:
+ spacja przed nawiasem

Reply:
ta przygoda nie będzie prosta: prosta -> łatwa

droga może być prosta, przygoda raczej nie... bo czy przygoda może być krzywa?

Reply:
i wgl to ostatnie zdanie to jest jakiś maturalny potwór, za długie! 🐉